python/MyriaPythonWorker.py

```diff
@@ -127,7 +127,6 @@ def read_command(self, stream):
         length = self.read_int(stream)
 
         if length < 0:
-            print("Command length is less than zero!", file=sys.stderr)
             raise ValueError("Command cannot have a length zero.")
```

Comment:
Again, the message is misleading since the exception is thrown when the length is _less than_ 0.